stdlib/distutils/debug.pyi

```diff
@@ -1 +1,3 @@
-DEBUG: bool | None
+from typing import Final
+
+DEBUG: Final[bool | None]
```

Comment:
I'd leave this to a followup PR

```suggestion
DEBUG: bool | None
```

Reply:
https://github.com/python/typeshed/pull/12332/commits/595204ade0fa7fb13ee44a18a6ff8e776512e615